src/pages/developers/tbtc-przegląd-techniczny-systemu-2.pl.md

```diff
@@ -0,0 +1,74 @@
+
+Ten model niesie ze sobą poważne konsekwencje wobec użytkowników i warto się z nim zaznajomić.
+
+Każdy depozyt musi odpowiadać jednemu ze standardowych wielkości lotów.
+
+System traktuje wszystkie przypadki nadpłaty i niedopłaty – użytkownik wpłaca kwotę większą lub mniejszą niż standardowa wielkość depozytu – jako niewłaściwe działanie użytkownika. Głównym skutkiem nadpłaty lub niedopłaty jest zniekształcenie zabezpieczenia sygnatariuszy. System ma za zadanie przerzucić wynikające z tego koszty na użytkownika.
+
+W przypadku niedopłaty – kiedy użytkownik wpłaci kwotę mniejszą niż wybrana wielkość lota BTC – system nie utworzy dowodu wpłaty, który można wymienić na TBTC. Użytkownik traci tym samym BTC zablokowane na depozycie, który może zostać podzielony między sygnatariuszy.
+
+Użytkownicy powinni być tego bardzo świadomi. W przypadku, gdy jedyną dostępną wielkością lota jest 1 BTC, łatwo sobie wyobrazić scenariusz, w którym użytkownik, który chce otrzymać 1 TBTC, dokonuje dwóch wpłat po 0,5 BTC każdy. Użytkownik, który to zrobi, straci wszystkie swoje BTC, ponieważ system odczyta to jako dwa przypadki niedopłaty. Krótko mówiąc, wielkość lota depozytu jest ustalana w momencie tworzenia depozytu i depozyt musi być zasilony tą kwota.
+
+W przypadku nadpłaty – kiedy użytkownik wpłaci więcej niż wybrana wielkość lota BTC – system wygeneruje dowód wpłaty, ale tylko w wysokości standardowej wielkości lota, z możliwością wykupu w zamian za tą kwotę w TBTC. Na rynku efektywnym spodziewalibyśmy się, że zostanie on natychmiast wykupiony, ponieważ wykupujący oczekuje, że nadpłacona kwota zablokowana na depozycie zostanie potraktowana jako arbitraż. O ile depozyt nie zostanie wykupiony przez pierwotnego deponenta, nadpłata przepada.
+
+Przykład: jeśli użytkownik zdeponuje 1,4 BTC w locie o wielkości 1 BTC, otrzyma on dowód pozwalający na wybicie dokładnie 1 TBTC (kwota odpowiadająca wielkości lota). W systemie pojawi się depozyt z nadpłatą, który inny użytkownik spodziewa się wykupić i wymienić 1 TBTC na 1,4 BTC. Użytkownik, który nadpłacił BTC będzie mógł, tak jak wszyscy inni użytkownicy, wymienić swój 1 TBTC na 1 BTC, ale dodatkowe 0,4 BTC zostanie utracone (chyba, że użytkownik zda sobie sprawę z popełnionego błędu i szybko wymieni swoje TBT na pierwotny depozyt 1,4 BTC).
+
+System zaakceptuje tylko UTXO większe niż wielkość lotu depozytu. Wszystkie inne BTC wysłane do grupy sygnatariuszy przepadają. Dlatego konieczne jest, aby deponenci wysyłali tylko jeden UTXO. Akceptowanie wielu UTXO od deponentów spowodowałoby znaczną złożoność łańcucha i opłaty za energie (ang. gas fees), ponieważ każde UTXO musiałoby zostać potwierdzone za pośrednictwem SPV i autoryzowanego podpisu. Sygnatariusze musieliby być zachęcani do podpisywania każdej transakcji, mimo, że całkowita wartość UTXO nie jest znana.
```

Comment:
*"wielkość lotu depozytu"* zmieniłbym na *"wielkość depozytu"*; tak jak wcześniej wspominałem, wydaje mi się że *lot size* nie ma dobrego tłumaczenia.

Reply:
*"System zaakceptuje tylko UTXO większe"* trzeba zamienić na "*System zaakceptuje tylko pierwsze UTXO większe"* - chodzi o to, że system leci po kolei po UTXO i bierze pierwsze z brzegu które pasuje do wymagań.

Reply:
"*i opłaty za energie*" chyba zamieniłbym po prostu na "*i opłaty*"; to "za energię" tutaj nie pasuje, to jest opłata dla minera za transakcje, niekoniecznie przeznaczy ją na rachunki za prąd :wink: